scipio/src/io/file_stream.rs

```diff
@@ -1392,15 +1374,6 @@ mod test {
         }
     }
 
-    file_stream_read_test!(read_close_twice, path, _k, file, _file_size: 131072, {
-        let mut reader = StreamReaderBuilder::new(file)
-            .with_buffer_size(1024)
-            .build();
-
-        reader.close().await.unwrap();
-        expect_specific_error(reader.close().await, "\"Bad file descriptor (os error 9)\"");
```

Comment:
This now doesn't compile -- one just can't close the file twice, it's gone after the first try.